src/Libraries/CoreNodeModels/Input/ColorPalette.cs

```diff
@@ -25,18 +29,46 @@ namespace CoreNodeModels.Input
     [OutPortNames("Color")]
     [OutPortTypes("Color")]
     [OutPortDescriptions("Selected Color.")]
-    public class ColorPalette : NodeModel
+    public class ColorPalette : NodeModel, INotifyPropertyChanged
```

Comment:
I don't think you need this interface as NodeModel should already implement INotifyPropertyChanged